src/daemon/ca/rc.rs

```diff
@@ -683,6 +706,37 @@ impl ResourceClass {
     }
 }
 
+/// # Autonomous System Provider Authorization
+///
+impl ResourceClass {
+    /// Renew all ROAs under the current for which the not-after time closer
```

Comment:
s/not-after time closer/not-after time is closer/ ?

Reply:
`/// Renew all ROAs` but the function name is `renew_aspas()`, i.e. this is not about ROAs?

Reply:
fixed